pysat/instruments/omni_hro.py

```diff
@@ -218,6 +218,9 @@ def calculate_imf_steadiness(inst, steady_window=15, min_window_frac=0.75,
 
     # Calculate the running circular standard deviation of the clock angle
     circ_kwargs = {'high': 360.0, 'low': 0.0}
+
+    # !!!!!!!!!!!!!!!!
+    # ca unused !!!!!!
```

Comment:
Looks like it could be dropped.

Not to self:  I also need to remove the older versions of these methods, along with my dmsp ones, from pysatDINEOFs.

Reply:
Since ca is unused, should we drop this line as well?  All subsequent lines use ca_std.

Reply:
I think it is fair to drop the line.

Reply:
pinging @aburrell since it is her method.

Reply:
These are used, look at line 229.

Reply:
I see ca_std being used, not ca.

Reply:
Confirmed that code tests not affected by dropping this line.  This function is tested in travis.